addons/account/models/account_bank_statement.py

```diff
@@ -732,8 +733,10 @@ def _prepare_move_line_default_vals(self, counterpart_account_id=None):
         '''
         self.ensure_one()
 
+        line_type = False
         if not counterpart_account_id:
             counterpart_account_id = self.journal_id.suspense_account_id.id
+            line_type = 'pay_counterpart_line'
```

Comment:
change the `counterpart_account_id` by a `**kwargs`, then
```py
        counterpart_vals = {
            'name': self.payment_ref,
            'account_id': counterpart_account_id,
            'amount_residual': liquidity_line_vals['debit'] - liquidity_line_vals['credit'],
            'line_type': line_type,
        }
```
becomes
```py
        counterpart_vals = {
            'name': self.payment_ref,
            'account_id': suspense_account.id,
            'amount_residual': liquidity_line_vals['debit'] - liquidity_line_vals['credit'],
           **kwargs,
        }
```
Then, you just need to call your method with `line_type = 'pay_counterpart_line'`. Also, I'm not sure it's a good idea to share the same line_type for both payments/statement lines